packages/test-generator/integration-test-templates/cypress/e2e/form/DSAllSupportedFormFields-spec.cy.ts

```diff
@@ -204,11 +205,123 @@ describe('FormTests - DSAllSupportedFormFields', () => {
       typeInAutocomplete(`Gr{downArrow}{enter}`);
       clickAddToArray();
 
+      const stringField = getInputByLabel('String');
+      stringField.should('have.value', 'Update1String');
+      stringField.type('X');
+      stringField.should('have.value', 'Update1StringX');
+
+      cy.contains('String1').should('exist');
+      getArrayFieldButtonByLabel('String array').click();
+      getInputByLabel('String array').type('String2');
+      clickAddToArray();
+      cy.contains('String2').should('exist');
```

Comment:
Is there something more specific we can assert on here? My concern is that if clicking add to array fails, there is still an input on screen with "String2" in it and this check would still pass.

Reply:
Yeah I’ll try to find something more specific. The important assertion for this functionality is on line 308

Reply:
If it's not a simple update to make, I'm okay with skipping this for now. The assertion on line 308 ensures that the add was successful.

Reply:
I changed it to this, and confirmed that if I add another string, the length is 2:
```
cy.contains('String2').siblings().should('have.length', 1);
```